packages/mui-material/src/styles/createTheme.test.js

```diff
@@ -5,10 +5,43 @@ import Button from '@mui/material/Button';
 import Box from '@mui/material/Box';
 import { ThemeProvider, createTheme, styled } from '@mui/material/styles';
 import { deepOrange, green } from '@mui/material/colors';
+import createPalette from './createPalette';
+
+const lightPalette = createPalette({ mode: 'light' });
+const darkPalette = createPalette({ mode: 'dark' });
 
 describe('createTheme', () => {
   const { render } = createRenderer();
 
+  it('should not have custom properties', () => {
```

Comment:
I think a more descriptive title would be

```suggestion
  it('should not create vars if cssVariables: false', () => {
```